sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernatePoolDao.java

```diff
@@ -201,13 +201,6 @@ public SessionFactory getSessionFactory() {
     return pools;
   }
 
-  @Override
-  public boolean remove(Pool t) throws IOException {
-    boolean exists = currentSession().get(PoolImpl.class, t.getId()) != null;
-    currentSession().delete(t);
-    return exists;
-  }
-
   @Override
   public void removeWatcher(Pool pool, User watcher) {
     log.debug("Removing watcher " + watcher.getLoginName() + " from " + pool.getWatchableIdentifier());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)